### PR TITLE
Translated Vietnamese UI

### PR DESCRIPTION
**Pull request by torn4dom4n**

### CODE REVIEW

**Review by trueberryless — APPROVED** [review]
LGTM, but I don't know if I'm allowed to approve changes...

File: .changeset/eighty-comics-pull.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@astrojs/starlight": patch
+---
+
+Translated Vietnamese UI.
```
Comment (delucis) [inline .changeset/eighty-comics-pull.md:5]: Tiny detail, but se’ve been using this format for changelogs recently, so would be nice to be consistent:

```suggestion
Updates Vietnamese UI translations
```

**Review by delucis — COMMENTED** [review]
Thank you for the quick update @torn4dom4n! Left one small suggestion just for the changeset.

And @trueberryless of course you’re more than welcome to leave reviews/approvals! It’s super helpful.

**Review by delucis — APPROVED** [review]
Thanks again for the translation! ✨

**Pull request merged**